src/tr2/game/shell/common.c

```diff
@@ -411,11 +412,11 @@ void Shell_Main(void)
     GameStringTable_Apply(nullptr);
 
     Savegame_Init();
+    GameBuf_Init();
```

Comment:
Can we move Savegame_Init() past Level_Init()?

Reply:
This new line in `Savegame_Init` needs to have executed first. Maybe it could be done lazily inside `Savegame_SetDefaultStats` instead?
`m_DefaultStats = Memory_Alloc(sizeof(STATS_COMMON) * GF_GetLevelTable(GFLT_MAIN)->count);`

Reply:
Sounds good.